src/masked-input/masked-input.style.tsx

```diff
@@ -1,21 +1,41 @@
 import styled from "styled-components";
 import { Color } from "../color";
+import { InputGroup } from "../input-group";
+
+interface InputGroupWrapperProps {
+    readOnly: boolean;
+    isDisabled: boolean;
```

Comment:
mark as a transient prop since this is not a html attribute

```suggestion
    $isDisabled: boolean;
```